js/form_validator.js

```diff
@@ -49,7 +49,24 @@ const EMAIL_INVALID = 'Please enter a correct email address format !';
 form.addEventListener('submit', (event) => {
   // validate the form
   const emailValid = validateEmail(form.elements.email, EMAIL_REQUIRED, EMAIL_INVALID);
+  const formData = new FormData();
+  const myObject = {};
   if (!emailValid) {
     event.preventDefault();
+  } else {
+    formData.append('name', form.elements.name.value);
+    formData.append('email', form.elements.email.value);
+    formData.append('message', form.elements.message.value);
+    formData.forEach((value, key) => {
+      myObject[key] = value;
+    });
+    localStorage.setItem('form-data', JSON.stringify(myObject));
```

Comment:
- [ ] Great job so far however, as requested by the previous reviewer, When the user changes the content of any input field in the form, the data object must be updated and saved to the local storage. even without having to submit the form, When the user loads the page, the input fields must be pre-filled with the data. kindly add this 👍🏻

Reply:
thank you! I resolved that.